<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加节点</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>节点名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="siteName">
        <property name="placeholderText">
         <string>关节点可不填</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>景色评分：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scoreComboBox">
          <item>
           <property name="text">
            <string>1分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10分</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>节点类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeComboBox">
          <item>
           <property name="text">
            <string>建筑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>景点</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>关节点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>节点坐标：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_lineEdit">
        <property name="placeholderText">
         <string>x坐标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y_lineEdit">
        <property name="placeholderText">
         <string>y坐标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>图上选点</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
